website/pages/docs/configuration/storage/raft.mdx

```diff
@@ -94,6 +94,16 @@ time. To use Raft for HA coordination users must also use Raft for storage.
   still need to be unsealed manually. See the section below that describes the
   parameters accepted by the `retry_join` stanza.
 
+- `kv_max_value_size` `(integer: 1572864)` - This configures the maximum number of
+  bytes for a value in a Raft entry. It applies to both Put operations and transactions.
+  Any value size in an entry or transaction exceeding this configuration value
+  will cause the respective operation to fail. Raft has a suggested max size of
+  data in a raft log entry. This is based on current architecture, default timing,
+  etc. Integrated storage also uses a chunk size that is the threshold used for
+  breaking a large value into chunks. By default, the chunk size is the same as
+  raft's max size log entry. The default value for this configuration is 1572864
```

Comment:
We may still want to discuss if this is the correct default size

Reply:
Thought about it some more and it seems like 3x is fine for now IMHO. Unless there are objections can we proceed with that?

Reply:
Per internal discussions, we've decided to have this value at 2x.